src/code/models/grid.ts

```diff
@@ -0,0 +1,72 @@
+import { types } from "mobx-state-tree";
+import { cellName, GridCell, IGridCell } from "./grid-cell";
+import { WeatherStation } from "./weather-station";
+import { v1 as uuid } from "uuid";
+
+const gridSizeInDegrees = 1;
+const defaultNumRows = 7;
+const defaultNumColumns = 7;
+
+function createCells(rows:number, columns:number) {
+  const cells:IGridCell[] = [];
+
+  for(let row = 0; row < rows; row++) {
+    for(let column = 0; column < columns; column++) {
+      const lon = (column * gridSizeInDegrees) + (0.5 * gridSizeInDegrees);
+      const lat = (row * gridSizeInDegrees) + (0.5 * gridSizeInDegrees);
+      const callSign = cellName(row,column);
+      const weatherStation = WeatherStation.create({
+        name: callSign,
+        callSign: callSign,
+        id: callSign,
+        imageUrl: "",
+        lat: lat,
+        long: lon
+      });
+      cells.push(GridCell.create( {
+        column: column,
+        row: row,
+        weatherStation: weatherStation
+      }));
+    }
+  }
+  return cells;
+}
+
+export const Grid = types.model('Grid', {
+  // Properties
+  id: types.optional(types.identifier(types.string), ()=> uuid()),
+  gridCells: types.optional(types.array(GridCell), () => createCells(this.rows, this.columns)),
+  columns: types.optional(types.number, defaultNumColumns),
+  rows: types.optional(types.number, defaultNumRows)
+}, {
+  // Colatile
```

Comment:
"Volatile", but it probably doesn't matter since it's going to go away soon anyway.